Documentation improvements (#145)
Stormbase/django-otp-webauthn: docs/how_to_guides/customize_models.rst

Customize models
==================

Django OTP WebAuthn provides the following base models for credentials
and attestations:

* ``AbstractWebAuthnCredential``
* ``AbstractWebAuthnAttestation``

These models have all the necessary fields and methods. If the base models don't
suit your needs, you can create custom models. However, if you're using a custom
model for credentials, you must also use a custom model for your attestation.

For more information, see the reference docs on :ref:`Models <models>`.

Create a custom model
---------------------

To create custom models, define a credential model that inherits from
``AbstractWebAuthnCredential``. Then, define an attestation model inheriting from
``AbstractWebAuthnAttestation``. Override the credential field in the attestation

> model to reference the custom credential model with a OneToOneField:

Reviewer: ```suggestion
model to reference the custom credential model with a ``OneToOneField``:
```

Handle database index name length errors
----------------------------------------

The ``AbstractWebAuthnCredential`` model automatically generates an index name
that includes the concrete model’s name, with ``_sha256_idx`` appended. If the
model name is more than 30 characters, the auto-generated index name may exceed

> your database's maximum identifier length. This can result in errors during migrations, such as:

Reviewer: ```suggestion
your database's maximum identifier length. This can result in errors during migrations. For example:
```
